Services/Randomization/test/ilRandomTest.php

```diff
@@ -0,0 +1,66 @@
+<?php declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+use ILIAS\DI\Container;
+use ILIAS\DI\LoggingServices;
+
+class ilRandomTest extends TestCase
+{
+    public function testConstruct() : void
+    {
+        $this->assertInstanceOf(\ilRandom::class, new ilRandom());
+    }
+
+    public function testIntWithoutArguments() : void
+    {
+        $random = new \ilRandom();
+
+        $random->int();
+        $this->assertTrue(true);
+    }
```

Comment:
Can we used something like ...

```suggestion
    public function testIntWithoutArguments() : void
    {
        $this->expectNotToPerformAssertions();
    
        $random = new \ilRandom();

         try {
                    $random->int();
        } catch (Error $e) {
            $this->fail('There should be not error when a random integer value is generated');
        }
    }
```

... instead?